nixos/modules/services/security/certmonger.nix

```diff
@@ -0,0 +1,89 @@
+{ pkgs, lib, config, ... }:
+with lib;
+let
+  cfg = config.services.certmonger;
+
+  cfgFile = pkgs.writeText "certmonger.conf" cfg.config;
+in
+{
+  options.services.certmonger = {
+    enable = mkEnableOption (mdDoc "certmonger service");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.certmonger;
+      defaultText = literalExpression "pkgs.certmonger";
+      description = mdDoc "certmonger package";
+    };
+
+    overrideHosts = mkOption {
+      type = types.bool;
+      default = true;
+      description = mdDoc ''
+        Remove /etc/hosts entries for "127.0.0.2" and "::1" defined in nixos/modules/config/networking.nix
+        networking.fqdn must be able to resolve their hostnames to their IP addresses, through PTR records
+        or /etc/hosts entries. Overwise IPA krb5 Keytab will try register as localhost.
+      '';
+    };
+
+    config = mkOption {
+      type = types.lines;
+      description = mdDoc "Contents of {file}`certmonger.conf`.";
+      default = ''
+        # This is the certmonger configuration file.  The format is a rather basic
+        # INI-style file.  See certmonger.conf(5) for notes about individual settings.
+        # · initial whitespace is ignored
+        # · whitespace between the key name and "=" is ignored
+        # · whitespace after "=" is ignored
+        # · trailing whitespace after values is ignored
+        # · comments begin with "#"
+        # · keys and section names are case-sensitive
+        # · there is no end-of-line continuation
+        #
+        # [defaults]
+        # notification_method = syslog
+        # notification_destination = daemon.notice
+        #
+        # [selfsign]
+        # validity_period = 1y
+        #
+        # [local]
+        # validity_period = 1y
+        #
```

Comment:
This should be rewritten to fit description instead imo

Reply:
done